internal/rbd/rbd_journal.go

```diff
@@ -278,7 +278,7 @@ func (rv *rbdVolume) Exists(ctx context.Context, parentVol *rbdVolume) (bool, er
 
 	// NOTE: Return volsize should be on-disk volsize, not request vol size, so
 	// save it for size checks before fetching image data
-	requestSize := rv.VolSize
+	requestSize := rv.VolSize //nolint:ifshort // FIXME: rename and split function into helpers
```

Comment:
```suggestion
	requestSize := rv.VolSize // nolint:ifshort // FIXME: rename and split function into helpers
```

Reply:
the convention is to have reason for nolint right after nolint comment. https://golangci-lint.run/usage/false-positives/
and `Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.`